api/data_refinery_api/views.py

```diff
@@ -584,6 +585,38 @@ def perform_update(self, serializer):
 
                 serializer.validated_data['is_processing'] = True
                 obj = serializer.save()
+
+                if settings.RUNNING_IN_CLOUD:
+                    try:
+                        try:
+                            remote_ip = get_client_ip(request)
+                            city = requests.get('https://ipapi.co/' + remote_ip + '/json/', timeout=10).json()['city']
+                        except Exception:
+                            city = ""
```

Comment:
Any chance this could be like "UNKNOWN" or something just so people don't think they found a bug when we can't determine the city?